CHANGELOG.md

```diff
@@ -1,26 +1,192 @@
+# 0.4.0 (November 5th, 2022)
+
+### Fixed
+
+- Fix panic in Deref/DerefMut for Slice extending into uninitialized part of the buffer ([#52])
+- by mzabaluev
+
+- chore: fix clippy lints ([#99])
+- by Noah-Kennedy
+
+- fix fs unit tests to avoid parallelism ([#121])
+- by FrankReh
+
+- Box the socket address to allow moving the Connect future ([#126])
+- by fasterthanlime
+
+- rt: Fix data race ([#146])
+- by ollie-etl
+
+
+### Added
+
+- Tokio-uring design proposal ([#1])
+- by carllerche
+
+- driver: batch submit requests and add benchmark ([#78])
+- by Noah-Kennedy
+
+- Implement fs::File::readv_at()/writev_at() ([#87])
+- by jiangliu
+
+- fs: implement FromRawFd for File ([#89])
+- by jiangliu
+
+- Implement `AsRawFd` for `TcpStream` ([#94])
+- by dpc
+
+- net: add TcpListener.local_addr method ([#107])
+- by FrankReh
+
+- write all ([#111])
```

Comment:
This one got mangled on merge. Let's put this in a format that makes it understandable.

Reply:
Made it net: add TcpStream.write_all